### DISCUSSION TITLE
[BUG] Ark server - [S_API FAIL] SteamAPI failed.

### DISCUSSION THREAD

**Discussion by Oreloth** [GameServerManagers/LinuxGSM#3405] [Q&A] [answered]
Error while starting fresh install of Ark server, all requirements are met.

```
Using binned.
4.5.1-0+UE4 7038 3077 404 10
[S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
Setting breakpad minidump AppID = 346110
```

Same error for non-LGSM Ark servers. Anyone has an idea to fix the SteamAPI error?

**Comment by wldwalker** [comment, marked as answer]
That's not an error. You can ignore it. It's just saying that the steam client isn't running. Which is fine since you are using it as a server.
